plugins/carla/cmake/carla-discovery-native.cmake

```diff
@@ -0,0 +1,35 @@
+add_executable(carla-discovery-native)
+mark_as_advanced(carla-discovery-native)
+
+set(carla_discovery_extra_libs ${carla_pthread_libs})
+
+if(OS_MACOS)
+  find_library(APPKIT AppKit)
+  mark_as_advanced(APPKIT)
+  set(carla_discovery_extra_libs ${carla_discovery_extra_libs} ${APPKIT})
+elseif(OS_WINDOWS)
+  set(carla_discovery_extra_libs ${carla_discovery_extra_libs} ole32 winmm)
```

Comment:
Use `library(APPEND <values>)` when adding values to a list, avoid using CMake 2.x style `set(<list> ${<list>} <new_values>)` pattern.

Reply:
no longer applies, marking as resolved